src/app/static/src/app/components/WarningAlert.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+    <div v-if="showAlert">
+        <div class="content alert alert-warning pt-0">
+            <div :style="containerBox">
+                <div ref="warningBox" id="warningBox">
+                    <div v-for="(value, key) in filteredWarnings" :key="key">
+                        <h4 class="alert-heading pt-2">
+                            <alert-triangle-icon size="1.5x" class="custom-class mr-1 mb-1"></alert-triangle-icon>
+                            {{ headerText(key) }}
+                        </h4>
+                        <ul class="mb-0">
+                            <li v-for="warning in value" :key="warning.text">{{ warning.text }}</li>
+                        </ul>
+                    </div>
+                </div>
+                <div ref="incHeader" class="invisible">
```

Comment:
What is `incHeader` short for? Spent a while trying to work out what this meant, so maybe a different name would be clearer!

Reply:
It meant 'including header' as it was a box containing a header and line plus any additional margin space in between. I'll just change it to `headerPlusLine` to be clearer.